src/EditorFeatures/TestUtilities/CodeActions/CSharpCodeFixVerifier`2.cs

```diff
@@ -72,5 +74,48 @@ public static async Task VerifyCodeFixAsync(string source, DiagnosticResult[] ex
             test.ExpectedDiagnostics.AddRange(expected);
             await test.RunAsync();
         }
+
+        public static async Task VerifyCodeFixAsync((string first, string second) sources, (string first, string second) fixedSources, int numberOfFixAllIterations)
+            => await VerifyCodeFixAsync(sources, fixedSources, DiagnosticResult.EmptyDiagnosticResults, numberOfFixAllIterations);
+
+        public static async Task VerifyCodeFixAsync((string first, string second) sources, (string first, string second) fixedSources, DiagnosticResult[] expected, int numberOfFixAllIterations)
```

Comment:
📝 The general approach to overloads for these verifier methods is avoid them unless they provide overwhelming benefit without being prone to combinatorial explosions. I don't believe the multiple sources scenario fits for this, and the `numberOfFixAllIterations` certainly wouldn't (it's only used for bugs).

Reply:
Can you suggest how to write this in VB?